c2corg_api/views/validation.py

```diff
@@ -162,7 +162,8 @@ def validate_token_pagination(request, **kwargs):
     Validate token pagination parameters (limit and token) for feed.
     """
     check_get_for_integer_property(request, 'limit', False)
-    validate_token(request)
+    simple_token = True if request.path == '/documents/changes' else False
```

Comment:
Checking for the request path is a bit fragile. If you change the route path, you'd have to remember to change it here.

Looking at `validate_token` I am wondering if it would not make sense to create a new function `validate_simple_token`. The checks for the simple and *complex* token have little in common after all.

And for `validate_token_pagination` to avoid checking the request path, you could either create a new function or have a signature like `validate_token_pagination(simple_token, request, **kwargs)` and use the validator like:

```python
@view(validators=[functools.partial(validate_token_pagination, True), ...
```